### ISSUE TITLE
Cell dragging results in cell deletion

### ISSUE THREAD

**Issue by Cryobus** [tahoma2d/tahoma2d#1610]
### Bug Description

When you select multiple cells across multiple Columns, and try to drag the bulk selection of the cells, the first leftmost cell that isnt selected gets removed for no reason, and there's no way to retrieve it, even with CTRL+Z.
Also, the operation fails to drag also the keyframes, leaving them in the previous position, resulting in messed up timeline (I have cell dragging options ticked in preferences saying to move cells and frame data).
![freecompress-TahomaError](https://github.com/user-attachments/assets/f7e340b1-ff73-4610-a480-414266e84ee2)

### Steps to reproduce

Just select multiple cells, even only across 1 column, make sure you hold SHIFT since you want to drag also the keyframes, and have at least 1 cell with a drawing behind the selection edge.
Drag the bulk selection and watch the frames just before the selection disappear

### Relevant logs/screenshots

![freecompress-TahomaError](https://github.com/user-attachments/assets/20b65b1a-0b1c-430e-951e-27f080f0ec5a)

### Software Version

Nightly (Specify below)

### Software Version Information

Aug 28-2024

### Operating System

Windows 10

### Graphics Card

NVIDIA GTX 1660 TI

### Drawing Tablet

XPPen

**Comment by Cryobus** [comment]
Of course these is critical since you want to change the timing often of your drawings, and adjust them; with this bug it's impossible to even do simple operations without the fear of inadvertedly lose a cell while you switch them around.

**Comment by manongjohn** [comment]
In order to move cells and keys at the same time, you need to 1st hold ALT (used to be CTRL but changed) and then do the selection of the cells and keys you want to move.  The keys will turn blue once selected so you know they will be included in the move.  When done with the selection, you can release the ALT key as it's no longer needed to move.

Holding SHIFT (or CTRL or ALT) while moving cells performs a special action when moving cells.  I believe SHIFT does an insert-type action causing the cells at the target drop cell to move to the end of the selection.

I will need to look to see why it caused cells to be lost like that, though.

**Comment by Cryobus** [comment]
Thank you very much for the explanation, I for some reason remembered the wrong key (Shift instead of ctrl) and discovered a bug in the meantime.

**Comment by Cryobus** [comment]
Alright I figured something else out, that's why I was having so many problems with cell dragging: there is something wrong with the key combination for dragging depending on the type of layout you have for the timeline. Here is what I found:

If in Preferences-Scene you have ticked Show Column and Cell Drag Bars:
You drag columns/cells by holding CTRL while selecting.

If in Preferences-Scene you have un-ticked Show Column and Cell Drag Bars:
You drag columns/cells by holding ALT while selecting.

This is the reason I was having so many problems, since I looked online and the instructions on how to move cells applied only when that box was ticked... but only because recently in tahoma it changed from CTRL to ALT, along with the option (which is default) to not show drag bars, but in reality this feature with the CTRL button is still inside Tahoma, but only if you decide to show drag bars. I think it might be a leftover from the switch from CTRL to ALT as the key dedicated to dragging.
